atomic_reactor/plugins/post_tag_and_push.py

```diff
@@ -95,11 +108,48 @@ def push_with_skopeo(self, registry_image, insecure, docker_push_secret):
             self.log.error("push failed with output:\n%s", e.output)
             raise
 
+    def source_get_unique_image(self):
+        source_result = self.workflow.prebuild_results.get(PLUGIN_FETCH_SOURCES_KEY)
+        koji_build_id = source_result['sources_for_koji_build_id']
+
+        kojisession = get_koji_session(self.workflow, NO_FALLBACK)
+        user_params = os.environ['USER_PARAMS']
+        user_data = json.loads(user_params)
+        container_target = user_data.get('koji_target')
```

Comment:
Is that the only way to get to it? It's a shame we don't store it somewhere after validating it in the input plugin.

Reply:
we don't store it in workflow, we are just using it plugins_configurations

Reply:
The way this normally seems to work is that we have the plugin take a `koji_target` parameter in its `__init__()` method, which osbs-client populates.

Reply:
sure I can do that and also create that osbs-client counterpart

Reply:
https://github.com/containerbuildsystem/osbs-client/pull/898

Work started here, I;d rather wait with this effort until that ^ PR is merged

Reply:
Maybe let's note this change for later then.

Reply:
added note

Reply:
using koji_target from __init__